client/app/components/ServicesTemplate/index.js

```diff
@@ -13,13 +13,15 @@ const ServicesTemplate = () => {
     <Card
       key={id}
       cardSize="is-6"
-      {...{ title,  content, url, img }}
+      img={<img role="presentation" src={require(`./images/${img}.svg`)} style={{ fill: '#000066'}} />}
+      {...{ title,  content, url}}
     />
   );
 
-  const filledCardServices = json.data.serviceCardContent.map(({ id, title, subtitle, url }) =>
+  const filledCardServices = json.data.serviceCardContent.map(({ id, title, subtitle, url, img }) =>
     <ServicesCardContent
       key={id}
+      img={<img role="presentation" src={require(`./images/${img}.svg`)} style={{ fill: '#000066'}} />}
       {...{ title, subtitle, url }}
```

Comment:
props should only ever be variable values, if they're static like in this case, they should be moved down to the child component (role, and style)

Reply:
You should also avoid passing down full elements down to children components, and only fill missing values.

Reply:
FYI @Globolobo this is a side effect of you originally making the shared Card work that way, just sayin' :heart: